qlty-cli/src/commands/coverage/complete.rs

```diff
@@ -69,8 +73,11 @@ impl Complete {
         print_authentication_info(&token, self.quiet);
 
         let timer = Instant::now();
-        Self::request_complete(&plan.metadata, &token).context("Failed to complete coverage")?;
-        self.print_complete_success(timer.elapsed().as_secs_f32());
+        let result = Self::request_complete(&plan.metadata, &token)
+            .context("Failed to complete coverage")?;
+
+        self.print_section_header(" MARKING COMPLETE...");
```

Comment:
I would simplify to just  `COMPLETING...`

Also I believe you need a trailing space in the string in order for the colored terminal output to shade the background color properly